internal/controller/bindata/webhook/14_server.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: network-resources-injector
+  name: network-resources-injector
+  namespace: {{.Namespace}}
+spec:
+  selector:
+    matchLabels:
+      app: network-resources-injector
+  template:
+    metadata:
+      labels:
+        app: network-resources-injector
+    spec:
+      serviceAccount: network-resources-injector-sa
+      containers:
+      - name: webhook-server
+        image: quay.io/vpunj/sriov-dp-admission-controller:latest
```

Comment:
Please change to quay.io/openshift/sriov-dp-admission-controller:latest

Reply:
can you add `origin-` to that url? quay.io/openshift/*origin-*sriov-dp-admission-controller

Reply:
Should this be dynamically set? Similar to how we set the daemon / vsp images, so we can [point to different images for development](https://github.com/openshift/dpu-operator/blob/main/config/dev/local-images-template.yaml), and have all production images specified [in a single place](https://github.com/openshift/dpu-operator/blob/main/config/manager/manager.yaml)?